frontend/src/data/runInfo.js

```diff
@@ -1,14 +1,5 @@
 import Summary from "./summary";
 
-const _DIALECT_URI_TO_SHORTNAME = {
-  "https://json-schema.org/draft/2020-12/schema": "Draft 2020-12",
```

Comment:
I actually missed reviewing this file when submitting my own PR. This code represents my previous attempts at structuring the data. As Julian pointed out, it would be beneficial to map the draft names to more readable and formatted ones. We can modify the code accordingly. No worries, I'll handle it in a separate PR.

Reply:
I missed your comment and already added the name mapping, hope you didn't start changing it on your end 😄